src/braket/experimental/autoqasm/converters/comparisons.py

```diff
@@ -0,0 +1,70 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+"""Converters for comparison nodes."""
```

Comment:
This grabs anything like `"a < b"` and replaces it with a function call that can later be hooked into and modified as we like.

In particular, `"a < b"` becomes `"ag__.lt_(a, b)"` which will then call the function `lt_` defined now in `operators/comparisons.py`